packages/osd-dev-utils/src/osd_client/osd_client_requester.ts

```diff
@@ -28,7 +28,7 @@
  * under the License.
  */
 
-import Url from 'url';
+import { URL } from 'url';
```

Comment:
Yes. We don't need to import the URL any more. It will automatically adapt in node.js and browser. The only one concern is node.js URL is different from browser URL and there may be some unexpected parsing differences.